pyscal/pyscalcli.py

```diff
@@ -165,8 +166,10 @@ def main():
             slgof=args.slgof,
             family2=args.family2,
         )
-    except ValueError:
-        # If ValueErrors, error messages have already been printed
+    except (OSError, ValueError) as err:
+        if args.debug:
+            print("".join(traceback.format_tb(err.__traceback__)))
+        print(str(err))
```

Comment:
I think you should only handle very specific exceptions here. If someone makes a programming error, the stacktrace is omitted if not `debug` is true? 😬

Reply:
Yes, I have thought of subclassing ValueError to a PyscalValueError or something and only catch that.

Reply:
`PyscalValueError` doesn't add anything in terms of meaning, though? We knew that the `ValueError` was in pyscal because we ran `pyscal`. That's probably something you don't want to do.

Reply:
It does not add meaning, but it would allow catching only that, and let ValueErrors that stem from programming errors to crash as usual with a unconditional traceback.

Reply:
Yes but why not both

Reply:
Not sure what you mean. Do you want to always print the stacktrace?

Reply:
No, I think you should hide the stacktrace when you handle a `PyscalValueError`, but what I mean is that `PyscalValueError` is a poor name for an exception class. 

For example, were you planning on doing this?

```python
raise PyscalValueError(
    f"You must supply low, base AND high, got only {set(remapped)}"
)
```

I think it would be _much_ better if you did this


```python
raise MissingMappingValuesError(
    f"You must supply low, base AND high, got only {set(remapped)}"
)
```

You can then catch this in main without accidentally catching actual programming/environmental errors.

Reply:
Ok! Redoing all exceptions is out of scope for this PR, but I will change back to always print stacktrace, and leave this for a future PR: